### ISSUE TITLE
Cell Editor: enable surface cleavage.

### ISSUE THREAD

**Issue by yakutovicha** [aiidalab/aiidalab-widgets-base#367]

**Comment by superstar54** [comment]
Sure, I will implement this.

**Comment by yakutovicha** [comment]
As far as I can tell this is not yet fixed.

**Comment by superstar54** [comment]
> As far as I can tell this is not yet fixed.

I agree. In a demo for aiidalab-qe structure editor plugin,  I did a test to cut the surface using [ase.build.surface](https://wiki.fysik.dtu.dk/ase/ase/build/surface.html#ase.build.surface). It works well, and is in the demo [repo](https://github.com/aiidalab/aiidalab-qe-plugin-demos/blob/main/aiidalab_qe_plugin_demos/surface/editor.py). I may make a PR to AWB when it's polished.
